src/APICalls/APICalls.js

```diff
@@ -91,4 +93,40 @@ export const deleteEvent = (eventId, playerId) => {
   .then(() => getAllEvents(playerId))
 }
 
+export const postFriendship = (followerId, followeeId) => {
+  return fetch(`${endpoints.friendshipDev}`, {
+    method: 'POST', 
+    body: JSON.stringify({
+      follower_id: followerId,
+      followee_id: followeeId
+    }),
+    headers: { 'Content-Type' : 'application/json' } 
+  })
+  .then(resp => {
+    if (resp.ok) {
+      return resp.json()
+    } else {
+      throw new Error('Unable to update friendship, please try again!')
+    }
+  })
+}
+
+export const deleteFriendship = (followerId, followeeId) => {
+  return fetch(`${endpoints.friendshipDev}`, {
+    method: 'DELETE',
+    body: JSON.stringify({
+      follower_id: followerId,
+      followee_id: followeeId
+    }),
+    headers: { 'Content-Type' : 'application/json' } 
+  })
+  .then(resp => {
+    if (resp.ok) {
+      return resp
```

Comment:
Stoked we got this all implemented...would've felt like a waste a couple days if we just had to delete all that code, nice!